docs/manual/common/concepts/ES_CQRS.md

```diff
@@ -1,6 +1,6 @@
 # Managing data persistence
 
-When designing your microservices, remember that each service should own its data and have direct access to the database. Other services should then use the Service API to interact with the data. There must be no sharing of databases across different services since that would result in a too-tight coupling between the services. In this way, each microservice defines a bounded context, similar to [Bounded Contexts](http://martinfowler.com/bliki/BoundedContext.html) in Domain-Driven Design.
+When designing your microservices, remember that each service should own its data and have direct access to the database. Other services should then use the Service API to interact with the data. There must be no sharing of databases across different services since that would result in a too-tight coupling between the services. In this way, each microservice operates within a clear boundary, which is known as a [Bounded Context](http://martinfowler.com/bliki/BoundedContext.html) in Domain-driven Design.
```

Comment:
I'm confused. The original says that the "clear boundary" is known as a bounded context -- to me that implies they are the same thing. Are they not? Or is it just that restricting other microservices from accessing the same DB as a particular microservice in itself is not a bounded context?